### PR TITLE
feat(connect-form): auto add advanced options and remove button

### PR DESCRIPTION
**Pull request by mcasimir**
https://user-images.githubusercontent.com/334881/151790771-5817bb1e-792c-46df-ba39-df9edcb89d3e.mp4

### CODE REVIEW

**Review by mabaasit — APPROVED** [review]

File: packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.spec.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ describe('UrlOptionsTable', function () {
 
   describe('with SearchParams', function () {
     beforeEach(function () {
+      cleanup();
```
Comment (gribnoysup) [inline packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.spec.tsx:33]: Why in the beforeEach and not afterEach?

File: packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.tsx [hunk 2]
```diff
@@ -164,7 +152,7 @@ function UrlOptionsTable({
               datum.name ? `${datum.name}-table-row` : 'new-option-table-row'
             }
           >
-            <Cell>
+            <Cell className={optionNameCellStyles}>
```
Comment (gribnoysup) [inline packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.tsx:155]: I asked this last time, but didn't get an answer: why are we using this table here at all? Seems like we are complicating layout both visually and from the code perspective, and the only thing this adds is the table headers that just duplicate the placeholders in the input fields

Comment (mcasimir) [reply packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.tsx:155]: Yeah, this makes sense probably ok to keep it for now

File: packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.spec.tsx (hunk 1 above)
Comment (mcasimir) [reply packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.spec.tsx:33]: looks like a leftover

**Pull request merged**